ci/publish-packages.sh

```diff
@@ -9,7 +9,7 @@ set -e -u -o pipefail
 function getSecrets() {
     if [[ ${BUILDKITE:-} ]]; then
 
-        export SPATIAL_OAUTH_DIR=$(mktemp -d)
+        export SPATIAL_OAUTH_DIR=$(mktemp -d "${TMPDIR:-/tmp}/XXXXXXXXX")
```

Comment:
These changes were for iterating on a MacOS machine, this still works on the Linux agents, but it just makes it easier in the future :)